docs/pages/admin-guides/access-controls/getting-started.mdx

```diff
@@ -1,193 +1,172 @@
 
-<Tabs>
-<TabItem label="SAML">
+You can use this labeling and matching logic to grant users specific access, ensuring they only interact with the resources they need.
 
-Follow our [SAML Okta Guide](./sso/okta.mdx) to
-create a SAML application.
+## Understanding Teleport’s YAML Configuration
 
-Save the file below as `okta.yaml` and update the `acs` field.
-Any member in Okta group `okta-admin` will assume a built-in role `admin`.
+Teleport’s roles are defined using YAML. Here’s a breakdown of a basic role configuration:
+
+- **kind**: The type of resource being created. For roles, this will always be role.
+- **metadata**: Includes information like the role’s name, which is used to identify the role.
+- **spec**: This section defines what the role can and cannot do.
+- **allow**: Lists the permissions the role grants, such as allowed logins or access to resources based on labels.
+- **deny**: Specifies any permissions that should be explicitly denied, even if they would otherwise be allowed.
+
+##  Creating a Basic Role for SSH Access
+
+- (!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+### Step 1/3
+
+Let’s set up a simple role that gives users access to staging servers for SSH access. Create a role definition file called staging-access.yaml with the following content:
 
 ```yaml
-kind: saml
-version: v2
+kind: role
+version: v7
 metadata:
-  name: okta
+  name: staging-access
 spec:
-  acs: https://tele.example.com/v1/webapi/saml/acs
-  attributes_to_roles:
-  - {name: "groups", value: "okta-admin", roles: ["access"]}
-  entity_descriptor: |
-    <?xml !!! Make sure to shift all lines in XML descriptor
-    with 4 spaces, otherwise things will not work
+  allow:
+    logins: ['dev']
+    node_labels:
+      'env': 'staging'
+  deny:
+    node_labels:
+      'env': 'production'
+
 ```
 
-Create the `saml` resource:
+### Step 2/3
+
+Apply this role using the following command:
 
 ```code
-$ tctl create okta.yaml
+$ tctl create -f staging-access.yaml
 ```
+This role allows the user to log in as the `dev` user to servers labeled with `env: staging` and explicitly denies access to any server labeled `env: production`.
 
-</TabItem>
-<TabItem label="OIDC">
+### Step 3/3
 
-Follow our [OIDC guides](./sso/oidc.mdx#identity-providers) to
-create an OIDC application.
+Assigning the Role to a User
 
-Copy the YAML below to a file called `oidc.yaml` and edit the information to
-include the details of your OIDC application.
+Now that you’ve created a role, assign it to a local user named Alice:
 
-```yaml
-(!examples/resources/oidc-connector.yaml!)
+```code
+$ tctl users add alice --roles=staging-access
 ```
 
-Create the `oidc` resource:
+Alice can now log in to staging servers, but she will be blocked from production environments.
+
+## Add local users with preset roles
```

Comment:
I'm not sure about the place of this section within the guide. Is it meant as a set of instructions or a conceptual description?

It's possible to map an SSO user to preset roles, so if we want to use this section to lay out the preset roles, I would make it about that and remove the local user aspect.